notificationbox/schema.json

```diff
@@ -34,6 +34,11 @@
             "description": "Text and icons for up to two notification action buttons.",
             "type": "array",
             "items": { "$ref": "Button" }
+          },
+          "css": {
+            "optional": true,
+            "description": "Additional CSS to apply to the notification.",
```

Comment:
"This allows you to e.g. set a custom colors. The style applies only the the notification bar element itself, not any child elements."

Reply:
```suggestion
            "description": "Additional CSS to apply to the notification. This allows you to e.g. set a custom colors. The style applies only the the notification bar element itself, not any child elements.",
```